fastlane/lib/fastlane/actions/update_info_plist.rb

```diff
@@ -80,7 +80,7 @@ def self.available_options
                                        description: "Path to info plist",
                                        optional: true,
                                        verify_block: proc do |value|
-                                         UI.user_error!("Invalid plist file") unless value[-6..-1].casecmp(".plist").zero?
+                                         UI.user_error!("Invalid plist file") unless value.end_with?(".plist")
```

Comment:
don't you want something like `value.downcase.end_with?(".plist")` to keep the `casecmp`? or it this not necessary anymore?

Alternatively `value =~ /\.plist^/i` is an option I guess

Reply:
Thank you for your review. applied suggestion. 🙇